.github/issue_template.md

```diff
@@ -0,0 +1,37 @@
+### Which version of the Ably SDK are you using? 
+E.g. `1.0.11`
+
+### On which platform does the issue happen? 
+E.g. `iOS11`
+
+### Are you using Carthage? 
+If yes, specify the version (`carthage version`)
+
+### Are you using Cocoapods? 
+If yes, specify the version (`pod --version`)
+
+### Which version of Xcode are you using? 
+(`xcodebuild -version`)
+
+### What did you do?
+Describe the problem.
+```
+Add a short snippet of code to show the problem
+```
+
+### What did you expect to happen?
+Describe the expected outcome.
+
+### What happened instead?
+Describe what happened instead.
+
+```
+Add relevant console logs.
+```
+
+**Tip:** set the log level to debug. E.g. `realtime.logger.logLevel = .debug`
```

Comment:
Should we perhaps recommend people first take a look at https://support.ably.io/solution/articles/3000063743-guide-information-we-need-to-help-you-diagnose-problems?  This may help them solve some problems themselves, especially when we improve the support section.

Additionally, should we not ask them for their app and key ID?

Reply:
@mattheworiordan about asking app ID see: https://ably-real-time.slack.com/archives/C030APSH3/p1522946396000289